wagtailcache/cache.py

```diff
@@ -116,11 +116,16 @@ def _chop_cookies(r: WSGIRequest) -> WSGIRequest:
     if not wagtailcache_settings.WAGTAIL_CACHE_IGNORE_COOKIES:
         return r
 
-    if r.COOKIES and not (
-        settings.CSRF_COOKIE_NAME in r.COOKIES
-        or settings.SESSION_COOKIE_NAME in r.COOKIES
-    ):
-        r.COOKIES = {}
+    r.COOKIES = {
+        k: v
+        for k, v in r.COOKIES.items()
```

Comment:
Thanks for doing this work! At CodeRed we avoid using lambdas if possible, because they can lead to code that is hard to read. Would you mind converting this to a traditional for-loop instead? Thanks!